integration_tests/debian_bd_integration_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./bd_integration_template.sh "debian" "mainline_kernel"
```

Comment:
I dont think this is a good idea for now, this would only complicate the run_integration_tests script and would make it harder to tailor test versions according to distros (for example a prestep for getting the .config file which follows a different process depending on the distro being used). I think we should keep it as is and once we have more tests with more distros we can make a more mature decision on if and how to refactor this parameters into the  run_integration tests script